.travis.yml

```diff
@@ -0,0 +1,24 @@
+
+dist: bionic
+sudo: false
+
+if: type = push
+
+env:
+  - TERRAFORM_DIR=main
+  - TERRAFORM_DIR=lambda
+
+# Install terraform
+before_install:
+  - curl -sLo /tmp/terraform.zip https://releases.hashicorp.com/terraform/0.12.19/terraform_0.12.19_linux_amd64.zip
+  - unzip /tmp/terraform.zip -d /tmp
+  - mkdir -p ~/bin
+  - mv /tmp/terraform ~/bin
+  - export PATH="~/bin:$PATH"
+  - export TF_INPUT=false
+  - export AWS_DEFAULT_REGION="us-east-1"
+
+script:
+  - cd $TERRAFORM_DIR
+  - terraform init
+  - terraform validate
```

Comment:
Use Travis build matrix to effectively run a pipeline that validates the main & lambda dirs in this repository